src/chart/generateCategoricalChart.tsx

```diff
@@ -839,6 +839,7 @@ export interface CategoricalChartProps {
   accessibilityLayer?: boolean;
   role?: string;
   tabIndex?: number;
+  defaultIndex?: number;
```

Comment:
Why is this a property of the chart? Could it be on the Tooltip instead?

Reply:
I scoped the property to the chart, since it's the chart that knows where the tooltip is, where the tickmarks are, what index the tooltip should be on, etc. Do you feel strongly that the property should live on the Tooltip?

Reply:
I have the same question - as a user I would expect to set a default tooltip index on the tooltip rather than the chart rather than the high level chart. The technical places where things live we can figure out, but the API should align with where someone would expect to see it

Reply:
Yes I'm thinking the same. From outside point of view, as a developer using Recharts, I have no knowledge of interest of the inner structure. If it's a property that controls where the Tooltip shows then that's a prop of Tooltip.

Or perhaps there are other use cases for this property that I am missing?

This decision is almost impossible to undo later so we need to get it right the first time

Reply:
Agreed, especially on the hard-to-undo part. Lets think this through..

Say we look here https://master--63da8268a0da9970db6992aa.chromatic.com/?path=/docs/api-chart-barchart--docs

The chart itself already has:
* `defaultShowTooltip` - this should be on Tooltip too TBH. Also how does this new prop interact with it?

On [Tooltip](https://master--63da8268a0da9970db6992aa.chromatic.com/?path=/docs/api-component-tooltip--docs) we have:
* `active` - this sets the the tooltip to visible always, how does this interact with the new prop? How does it interact with defaultShowTooltip? This is already confusing I think...

Reply:
From what I can tell `defaultShowTooltip` does't work...